### ISSUE TITLE
emailServer() is holding a write lock on server.sqlite

### ISSUE THREAD

**Issue by jbrzusto** [jbrzusto/motusServer#69]
why?  Was there a leftover -journal file the last time it was run?
Or is it that calls like these:

-   processServer:         
```R
handled = loggingTry(j, h(j))
```
and

-  emailServer:  
```R
loggingTry(j, motusHandled <<- h(j))
```

are somewhere starting a transaction on the first `j` then running stuff with
the second `j`?

Look at the use of savepoints in Copse.R and lazy eval for situations like
```R
j$XXX <- f(j)
```
If the right side takes a long time, that's probably what we're running into.
These use cases need to be rethought or eliminated, e.g. by always
assigning f(j) to a temporary intermediate.  But better to fix the savepoint logic...

**Comment by jbrzusto** [comment]
logging Copse savepoint / release / rollback shows a long-lived savepoint generated by
`setData.Twig()`, immediately followed by many balanced pairs of savepoint / release

**Comment by jbrzusto** [comment]
*should* be closed by a71f0f44b797597805169a98d7ebcee7fbca9e5a
